### PR TITLE
Updated README.

### LINKED ISSUE #991: pip install ocean-lib not working on Ubuntu Linux with Python 3.10.6
**Describe the bug**
ocean-lib is not installing correctly on Linux (Ubuntu 22.04.1) with Python 3.10.6

I followed the installation described in the Simple Remote Flow (https://github.com/oceanprotocol/ocean.py/blob/main/READMEs/simple-remote.md).

running the following commands:
```
cd ~
mkdir my_project
cd my_project
python3 -m venv venv
source venv/bin/activate
pip3 install wheel
pip3 install ocean-lib
```
throws this error

```
Collecting varint
  Downloading varint-1.0.2.tar.gz (1.9 kB)
  Preparing metadata (setup.py) ... done
Building wheels for collected packages: PyYAML, coincurve, websockets, parsimonious, varint
  Building wheel for PyYAML (pyproject.toml) ... done
  Created wheel for PyYAML: filename=PyYAML-5.4.1-cp310-cp310-linux_x86_64.whl size=45656 sha256=bfc68c6e067c06863426be0737a9401224fd045b61e4727999b76efd81b954c3
  Stored in directory: /home/mihai/.cache/pip/wheels/c7/0d/22/696ee92245ad710f506eee79bb05c740d8abccd3ecdb778683
  Building wheel for coincurve (pyproject.toml) ... error
  error: subprocess-exited-with-error
  
  × Building wheel for coincurve (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [66 lines of output]
      running bdist_wheel
      running build
      running build_py
      creating build
      creating build/lib.linux-x86_64-cpython-310
      creating build/lib.linux-x86_64-cpython-310/coincurve
      copying coincurve/context.py -> build/lib.linux-x86_64-cpython-310/coincurve
      copying coincurve/keys.py -> build/lib.linux-x86_64-cpython-310/coincurve
      copying coincurve/utils.py -> build/lib.linux-x86_64-cpython-310/coincurve
      copying coincurve/__init__.py -> build/lib.linux-x86_64-cpython-310/coincurve
      copying coincurve/ecdsa.py -> build/lib.linux-x86_64-cpython-310/coincurve
      copying coincurve/flags.py -> build/lib.linux-x86_64-cpython-310/coincurve
      copying coincurve/types.py -> build/lib.linux-x86_64-cpython-310/coincurve
      copying coincurve/_windows_libsecp256k1.py -> build/lib.linux-x86_64-cpython-310/coincurve
      copying coincurve/py.typed -> build/lib.linux-x86_64-cpython-310/coincurve
      running build_clib
      /tmp/pip-install-nrlvv3il/coincurve_2064f308d9ca470a80254a6fcb6edd41/libsecp256k1/autogen.sh: 3: autoreconf: not found
      Traceback (most recent call last):
        File "/home/mihai/my_project/venv/lib/python3.10/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 363, in <module>
          main()
        File "/home/mihai/my_project/venv/lib/python3.10/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 345, in main
          json_out['return_val'] = hook(**hook_input['kwargs'])
        File "/home/mihai/my_project/venv/lib/python3.10/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 261, in build_wheel
          return _build_backend().build_wheel(wheel_directory, config_settings,
        File "/tmp/pip-build-env-690gpljt/overlay/lib/python3.10/site-packages/setuptools/build_meta.py", line 412, in build_wheel
          return self._build_with_temp_dir(['bdist_wheel'], '.whl',
        File "/tmp/pip-build-env-690gpljt/overlay/lib/python3.10/site-packages/setuptools/build_meta.py", line 397, in _build_with_temp_dir
          self.run_setup()
        File "/tmp/pip-build-env-690gpljt/overlay/lib/python3.10/site-packages/setuptools/build_meta.py", line 482, in run_setup
          super(_BuildMetaLegacyBackend,
        File "/tmp/pip-build-env-690gpljt/overlay/lib/python3.10/site-packages/setuptools/build_meta.py", line 335, in run_setup
          exec(code, locals())
        File "<string>", line 260, in <module>
        File "/tmp/pip-build-env-690gpljt/overlay/lib/python3.10/site-packages/setuptools/__init__.py", line 87, in setup
          return distutils.core.setup(**attrs)
        File "/tmp/pip-build-env-690gpljt/overlay/lib/python3.10/site-packages/setuptools/_distutils/core.py", line 185, in setup
          return run_commands(dist)
        File "/tmp/pip-build-env-690gpljt/overlay/lib/python3.10/site-packages/setuptools/_distutils/core.py", line 201, in run_commands
          dist.run_commands()
        File "/tmp/pip-build-env-690gpljt/overlay/lib/python3.10/site-packages/setuptools/_distutils/dist.py", line 968, in run_commands
          self.run_command(cmd)
        File "/tmp/pip-build-env-690gpljt/overlay/lib/python3.10/site-packages/setuptools/dist.py", line 1217, in run_command
          super().run_command(command)
        File "/tmp/pip-build-env-690gpljt/overlay/lib/python3.10/site-packages/setuptools/_distutils/dist.py", line 987, in run_command
          cmd_obj.run()
        File "<string>", line 95, in run
        File "/tmp/pip-build-env-690gpljt/overlay/lib/python3.10/site-packages/wheel/bdist_wheel.py", line 299, in run
          self.run_command('build')
        File "/tmp/pip-build-env-690gpljt/overlay/lib/python3.10/site-packages/setuptools/_distutils/cmd.py", line 319, in run_command
          self.distribution.run_command(command)
        File "/tmp/pip-build-env-690gpljt/overlay/lib/python3.10/site-packages/setuptools/dist.py", line 1217, in run_command
          super().run_command(command)
        File "/tmp/pip-build-env-690gpljt/overlay/lib/python3.10/site-packages/setuptools/_distutils/dist.py", line 987, in run_command
          cmd_obj.run()
        File "/tmp/pip-build-env-690gpljt/overlay/lib/python3.10/site-packages/setuptools/_distutils/command/build.py", line 132, in run
          self.run_command(cmd_name)
        File "/tmp/pip-build-env-690gpljt/overlay/lib/python3.10/site-packages/setuptools/_distutils/cmd.py", line 319, in run_command
          self.distribution.run_command(command)
        File "/tmp/pip-build-env-690gpljt/overlay/lib/python3.10/site-packages/setuptools/dist.py", line 1217, in run_command
          super().run_command(command)
        File "/tmp/pip-build-env-690gpljt/overlay/lib/python3.10/site-packages/setuptools/_distutils/dist.py", line 987, in run_command
          cmd_obj.run()
        File "<string>", line 152, in run
        File "/usr/lib/python3.10/subprocess.py", line 369, in check_call
          raise CalledProcessError(retcode, cmd)
      subprocess.CalledProcessError: Command '['/tmp/pip-install-nrlvv3il/coincurve_2064f308d9ca470a80254a6fcb6edd41/libsecp256k1/autogen.sh']' returned non-zero exit status 127.
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for coincurve
  Building wheel for websockets (setup.py) ... done
  Created wheel for websockets: filename=websockets-9.1-cp310-cp310-linux_x86_64.whl size=85892 sha256=a8594dca6d309e7d1790fd7c794c928be95ad645455fe00e460e036e154e18a2
  Stored in directory: /home/mihai/.cache/pip/wheels/79/f7/4e/873eca27ecd6d7230caff265283a5a5112ad4cd1d945c022dd
  Building wheel for parsimonious (setup.py) ... done
  Created wheel for parsimonious: filename=parsimonious-0.8.1-py3-none-any.whl size=42723 sha256=e130a16c0bbe4d2ac4519684f4a0f7c634115ec10894dde55043d6c5eb92915c
  Stored in directory: /home/mihai/.cache/pip/wheels/b1/12/f1/7a2f39b30d6780ae9f2be9a52056595e0d97c1b4531d183085
  Building wheel for varint (setup.py) ... done
  Created wheel for varint: filename=varint-1.0.2-py3-none-any.whl size=1980 sha256=cf7bfd5013ab8a020559e2a3b1374f6e04adeae322a0ac96f719880c2d161632
  Stored in directory: /home/mihai/.cache/pip/wheels/39/48/5e/33919c52a2a695a512ca394a5308dd12626a40bbcd288de814
Successfully built PyYAML websockets parsimonious varint
Failed to build coincurve
ERROR: Could not build wheels for coincurve, which is required to install pyproject.toml-based projects
```

**To Reproduce**

1. Install latest Ubuntu version (22.04.1)
```
2. sudo apt install python3.10-venv
3. cd ~
4. mkdir my_project
5. cd my_project
6. python3 -m venv venv
7. source venv/bin/activate
8. pip3 install wheel
9. pip3 install ocean-lib
```

**Expected behavior**
ocean-lib is installed successfully.

**Desktop (please complete the following information):**
 - OS: Ubuntu 22.04.1 running on VMWare Workstation 16 Player 
 - Python: 3.10.6 (preinstalled by OS installation)

### PR DESCRIPTION
**Pull request by mariacarmina**
Fixes #991  .

Changes proposed in this PR:

- updated python version in the simple flow README
- mentioned the issue for M1 type processors also in the simple flow README.

### CODE REVIEW

**Review by trentmc — CHANGES_REQUESTED** [review]
This is way too verbose. Figure out a way to make it a one-liner. If needed, link to a separate readme with details.

File: READMEs/data-nfts-and-datatokens-flow.md [hunk 1]
```diff
@@ -52,6 +52,19 @@ pip3 install wheel
 pip3 install --pre ocean-lib
 ```
 
+For M1 processors, if the installation of ocean lib fails due to `coincurve` or
```
Comment (calina-c) [inline READMEs/data-nfts-and-datatokens-flow.md:55]: I would create a subsection or a different readme called "Known issues" and include a shortened version of this content. How about that? @trentmc @mariacarmina

**Review by MBadea17 — COMMENTED** [review]
Currently, there are two different sections of the installation of ocean-lib. One is in the main readme (https://github.com/oceanprotocol/ocean.py#-installation), the other one is in the simple flow (https://github.com/oceanprotocol/ocean.py/blob/main/READMEs/data-nfts-and-datatokens-flow.md#install-the-library). Also, the descriptions are slightly different (one is in Python virtual environment, the other is without venv), which is confusing and adds friction. I would leave it in a single place - in the main read.me.

Agree with Calina to have a section for known issues.

**Comment by calina-c** [comment]
Integrated in ocean brownie branch

**Pull request closed without merging**